tests/test_sql_validator.py

```diff
@@ -141,12 +141,28 @@ def test_extract_error_details_error_non_str_message_details(validator, project)
         validator._extract_error_details(query_result)
 
 
-def test_extract_error_details_error_loc_wo_msg_details(validator, project):
-    message = "An error message."
-    sql = "SELECT * FROM orders"
+def test_extract_error_details_for_unset_access_filter(validator, project):
```

Comment:
Can we make this more of a generalized test case title? That is, what's the combination of fields that's being tested here? `message` is present but `message_details` is not? Is it that `sql` is not present? Want to give the test a title that describes what's being tested in an abstract sense rather than the exact error that you received.

Reply:
Also, I believe we can remove the fixture `project` here since I don't see it being used.

Reply:
Updated, let me know if there's anything else